.idea/flexy-thingworx-connector.iml

```diff
@@ -12,6 +12,6 @@
     <orderEntry type="sourceFolder" forTests="false" />
     <orderEntry type="library" scope="TEST" name="Maven: junit:junit:3.8.1" level="project" />
     <orderEntry type="library" scope="PROVIDED" name="Maven: com.hms_networks.americas.sc.mvnlibs:ewon-etk:1.4.4" level="project" />
-    <orderEntry type="library" name="Maven: com.hms_networks.americas.sc:extensions:1.5.0" level="project" />
+    <orderEntry type="library" name="Maven: com.hms_networks.americas.sc:extensions:1.6.0" level="project" />
```

Comment:
Are you using IntelliJ now?

Reply:
no. I searched for 1.5.0 and updated everything I found.

Reply:
Ah, ok